homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/styles/main.css

```diff
@@ -0,0 +1,84 @@
+main {
+  margin-left: 53px;
+  margin-right: 52px;
+}
+
+/* About Page*/
+
+.about__container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 68px;
+  align-items: start;
+  margin-top: 140px;
```

Comment:
Please correct the margin according to design 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/d73d3922-a894-4277-8f2d-c96928001028)

Reply:
still seeing this